src/SourceBuild/content/build.proj

```diff
@@ -34,6 +34,7 @@
 
     <Message Text="Build Mode: $(BuildModeInfoText)" Importance="high" />
     <Message Text="Build Environment: $(TargetArchitecture) $(Configuration) $(TargetOS) $(TargetRid)" Importance="high" />
+    <Message Condition="'$(EnablePoison)' == 'true'" Text="Crossgen is disabled for poisoning" Importance="high" />
```

Comment:
I would rephrase this slightly to something like "Crossgen is disabled because it's in-compatible when performing poison checks.".

Reply:
Addressed in https://github.com/dotnet/sdk/pull/42228/commits/f6cd9d3c9b3fd82df237577170d79b75bdf2cd59